assets/prototype/domain/eventEditor/containers/queries/useInitQueries.js

```diff
@@ -27,24 +25,18 @@ const useInitQueries = ({ eventId }) => {
 		},
 	});
 
-	if (!initialized) {
-		toaster.loading(loadingDates, toasterDatesMessage);
-		toaster.error(datetimeError);
-	}
-
 	const datetimes = pathOr([], ['datetimes', 'nodes'], datetimesData);
 	const datetimeIn = datetimes ? datetimes.map(({ id }) => id) : [];
-
 	const toasterTicketsMessage = 'initializing event editor tickets';
-	const [
-		fetchRelatedTickets,
-		{ data: ticketsData, error: ticketError, loading: loadingTickets, called },
-	] = useLazyQuery(GET_TICKETS, {
+	const skipTicketsQuery = datetimeIn.length === 0;
+
+	const { data: ticketsData, error: ticketError, loading: loadingTickets } = useQuery(GET_TICKETS, {
 		variables: {
 			first: 50,
 			where: {
 				datetimeIn,
 			},
+			skip: skipTicketsQuery,
```

Comment:
`skip` is not to be added to variables, it should be added to options in `useMutation`.

Reply:
:man_facepalming:

Reply:
thx :)